cookbooks/bcpc-hadoop/recipes/disks.rb

```diff
@@ -1,16 +1,21 @@
+# vim: tabstop=2:shiftwidth=2:softtabstop=2 
```

Comment:
Sets up spacing for vim users, emacs users are welcome to do so as well.  Good for enforcing code standard.